Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupThemeComboBox.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
```

Comment:
Remove these into Global
And add License (Just Krypton, Not Component)

Reply:
Resolved